### PR TITLE
13721 Add text and hyperlink to Simplified Search home page

### PR DESCRIPTION
**Pull request by nitu14598**
Hi,

Added code changes and unit test cases
Please review and merge the changes

Regards!

### CODE REVIEW

File: src/app/features/fss-search/fss-simplified-search/fss-simplified-search.component.scss [hunk 1]
```diff
@@ -28,4 +28,27 @@
...
-  }
\ No newline at end of file
+}
+.switchToESS{
+	width: 100%; 
+	float: right;
+    text-align: right;
+	padding: 0px;
+}
+	.Exchangesetpara{
```
Comment (crdandoukho) [inline src/app/features/fss-search/fss-simplified-search/fss-simplified-search.component.scss:38]: Please follow the existing naming convention for all the new CSS classes (camel case)

For example: `.Exchangesetpara` should be `.exchangeSetPara`

File: src/app/features/fss-search/fss-simplified-search/fss-simplified-search.component.scss [hunk 2]
```diff
@@ -28,4 +28,27 @@
...
+		font-size: 17px;
+	}
+
+  .Searchmainaindiv{
```
Comment (crdandoukho) [inline src/app/features/fss-search/fss-simplified-search/fss-simplified-search.component.scss:42]: I think there's a typo in the name of the `.Searchmainaindiv` class. Should it be `.Searchmaindiv` (`.searchMainDiv`)?

File: playwright.config.ts [hunk 3]
```diff
@@ -48,19 +48,19 @@ const config: PlaywrightTestConfig = {
     //   },
     // },
 
-    {
-      name: 'webkit',
-      use: {
-        ...devices['Desktop Safari'],
-      },
-    },
+    // {
```
Comment (crdandoukho) [inline playwright.config.ts:51]: Please confirm that this commented config is intended to be merged into main

**Review by crdandoukho — CHANGES_REQUESTED** [review]
A few simple changes needed please

File: playwright.config.ts (hunk 3 above)
Comment (crdandoukho) [reply playwright.config.ts:51]: Kishor has confirmed the commented code is intentional

So I will resolve this conversation

File: src/app/features/fss-search/fss-simplified-search/fss-simplified-search.component.scss (hunk 1 above)
Comment (nitu14598) [reply src/app/features/fss-search/fss-simplified-search/fss-simplified-search.component.scss]: resolved by adding changes

File: src/app/features/fss-search/fss-simplified-search/fss-simplified-search.component.scss (hunk 2 above)
Comment (nitu14598) [reply src/app/features/fss-search/fss-simplified-search/fss-simplified-search.component.scss]: Resolved by doing changes

**Review by crdandoukho — APPROVED** [review]
Looks good

**Review by rvijayavel — APPROVED** [review]
Looks good.

**Pull request merged**